packages/js/src/ui/components/Notification/DefaultNotification.tsx

```diff
@@ -11,20 +20,93 @@ export const DefaultNotification = (props: DefaultNotificationProps) => {
   const style = useStyle();
 
   return (
-    <div class={style('notificationContainer', 'nt-w-full nt-p-6')}>
-      <div class={style('notificationSubjectContainer', 'nt-flex nt-items-center nt-gap-2 nt-relative')}>
-        <span
+    <div
+      class={style(
+        'notification',
+        'nt-w-full nt-text-sm hover:nt-bg-neutral-100 nt-group nt-relative nt-flex nt-px-6 nt-py-4 nt-gap-2'
+      )}
+    >
+      <span
+        class={style(
+          'notificationDot',
+          'nt-absolute -nt-translate-x-[150%] nt-translate-y-1/2 nt-size-2.5 nt-bg-primary nt-rounded-full nt-border'
+        )}
+      />
+      <Show when={props.notification.avatar}>
+        <img class={style('notificationImage', 'nt-size-8 nt-rounded-lg')} src={props.notification.avatar} />
+      </Show>
+      <div class={style('notificationBody', 'nt-overflow-hidden nt-w-full')}>
+        <p
           class={style(
-            'notificationDot',
-            'nt-absolute -nt-left-3 nt-top-0 -nt-translate-x-1/2 nt-translate-y-1/2 nt-size-3 nt-bg-primary nt-rounded-full nt-border'
+            'notificationDate',
+            'nt-text-foreground-alpha-400 nt-shrink-0 nt-float-right group-hover:nt-hidden'
           )}
-        />
+        >
+          {/* TODO: pass locale here */}
+          {formatToRelativeTime({ fromDate: new Date(props.notification.createdAt) })}
+        </p>
+        <div
+          class={style(
+            'notificationDefaultActions',
+            'nt-gap-3 nt-shrink-0 nt-float-right nt-hidden group-hover:nt-flex'
+          )}
+        >
+          <Show
+            when={props.notification.isRead}
+            fallback={
+              <Button size="icon" variant="icon">
+                <ReadAll />
+              </Button>
+            }
+          >
+            <Button size="icon" variant="icon">
+              <Archive />
+            </Button>
+          </Show>
+          <Show
+            when={props.notification.isArchived}
+            fallback={
+              <Button size="icon" variant="icon">
+                <Archive />
+              </Button>
+            }
+          >
+            <Button size="icon" variant="icon">
+              <Unarchive />
+            </Button>
+          </Show>
+        </div>
+        <Show when={props.notification.subject} fallback={<NotificationBody>test</NotificationBody>}>
+          <p
+            class={style(
+              'notificationSubject',
+              'nt-font-semibold nt-overflow-ellipsis nt-whitespace-nowrap nt-overflow-hidden'
+            )}
+          >
```

Comment:
✏ **suggestion:** please use the `title` attribute to show the value when it's cropped, same for the `body`